packages/hardhat-verify/src/internal/sourcify.ts

```diff
@@ -14,14 +14,23 @@ import { ValidationResponse } from "./utilities";
 export class Sourcify {
   public apiUrl: string = "https://sourcify.dev/server";
   public browserUrl: string = "https://repo.sourcify.dev";
+  private _chainId: number;
```

Comment:
this change doesn't seem necessary for this PR. Please revert it.

Reply:
The linter forced me to do it

Reply:
I just tested it locally, and you should be able to keep `chainId` public without linting errors. Maybe it's a problem in your config?

![image](https://github.com/NomicFoundation/hardhat/assets/1165424/428be6fd-3c9e-4fcf-8d94-d3b8ca90889b)
![image](https://github.com/NomicFoundation/hardhat/assets/1165424/96f1e197-e2cc-48ca-acbd-ef7e4a06a99c)

Reply:
No ok sorry, I thought you were referring to the underscore part. I reverted the change